docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -555,6 +555,12 @@ To send a command to the server from the client side, follow these steps:
    mc_client_send_shuffle_mode_cmd(g_client_h, server_name, MC_SHUFFLE_MODE_ON, NULL);
    mc_client_send_repeat_mode_cmd(g_client_h, server_name, MC_REPEAT_MODE_OFF, NULL);
    mc_client_send_playlist_cmd(g_client_h, server_name, "my_favorite", "1", MC_PLAYBACK_ACTION_PLAY, 0, NULL);
+   
+   Since Tizen 5.5, the following APIs are supported:
```

Comment:
Would you please revert back this sentence to "/* Since Tizen 5.5, following APIs are supported */ "?

This line is in the sample code block. It is really a comment line for the code.

![image](https://user-images.githubusercontent.com/9958374/67074725-a2e03f00-f1c4-11e9-9142-53edae04f02a.png)


Please don't be confused with similar sentence on line 614. 
Only fix to the line 559 is required.

Reply:
Done